src/gui/folder.cpp

```diff
@@ -1144,20 +1144,27 @@ SyncOptions Folder::initializeSyncOptions() const
 {
     SyncOptions opt;
     ConfigFile cfgFile;
+    const auto account = _accountState->account();
 
     auto newFolderLimit = cfgFile.newBigFolderSizeLimit();
     opt._newBigFolderSizeLimit = newFolderLimit.first ? newFolderLimit.second * 1000LL * 1000LL : -1; // convert from MB to B
     opt._confirmExternalStorage = cfgFile.confirmExternalStorage();
     opt._moveFilesToTrash = cfgFile.moveToTrash();
     opt._vfs = _vfs;
-    opt._parallelNetworkJobs = _accountState->account()->isHttp2Supported() ? 20 : 6;
+    opt._parallelNetworkJobs = account->isHttp2Supported() ? 20 : 6;
```

Comment:
should not this be modified to read the new capability provided by https://github.com/nextcloud/server/pull/48758/files#diff-1f982797e6086d996b7354b65b1f5384b902ffe98da4a9dd121ff1a2f33e6252R39 ?

Reply:
good point, I had missed this